github/data_source_github_actions_environment_public_key_test.go

```diff
@@ -0,0 +1,49 @@
+package github
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccGithubActionsEnvironmentPublicKeyDataSource(t *testing.T) {
+
+	t.Run("queries an environment public key without error", func(t *testing.T) {
+
+		config := `
```

Comment:
Currently, this test will not run as the configuration is missing the required environment and repository parameters. You can look at `github/data_source_github_actions_environment_secrets_test.go` for an example of how to construct a repo and environment in a test case.